OpenSim/Simulation/SimbodyEngine/Joint.h

```diff
@@ -226,14 +226,12 @@ OpenSim_DECLARE_ABSTRACT_OBJECT(Joint, ModelComponent);
     // Coordinate Set
     const CoordinateSet& getCoordinateSet() const {return get_CoordinateSet();}
```

Comment:
Consider removing this (redundant) accessor, since we: 1) do not want to promote access to individual Coordinates that bypasses the less error prone Coord enum interface , and 2) want to make it easier to convert to using a fixed sized property list.  Can be done as subsequent PR (just noting here so I don't forget).